### PR TITLE
Feature/fix eslint

### PR DESCRIPTION
**Pull request by chemoish**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -1,33 +1,39 @@
-module.exports = function (config) {
-    config.set({
-        browsers: ['PhantomJS'],
+module.exports = (config) => {
```
Comment (gabehayes) [inline karma.conf.js:1]: If you are using ES6, should you just `export default` this? Also, it looks like Travis is not responding properly to the ES6 change here, is there something you need to update to make that work?

Comment (chemoish) [reply karma.conf.js:1]: Branches are based off other branches, the node version got updated in the next chain which allows things to work.

File: src/menu/chapter-thumbnail-menu-item.js [hunk 2]
```diff
@@ -15,58 +17,60 @@ import template from '../videojs-chapter-thumbnail-template';
...
+      textTrack,
+    } = options;
 
-        options.label    = template(cue);
-        options.selected = (cue.startTime <= current_time && current_time < cue.endTime);
+    const currentTime = player.currentTime();
 
-        super(player, options);
+    super(player, {
+      ...options,
```
Comment (chemoish) [inline src/menu/chapter-thumbnail-menu-item.js:29]: options are kinda confusing, is there better way?

Comment (chemoish) [reply src/menu/chapter-thumbnail-menu-item.js:29]: /shrug

**Pull request merged**